src/Compilers/CSharp/Test/Emit/PrivateProtected.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#if NET461
```

Comment:
I'm not a fan of this approach vs ConditionalFact, as we'll miss these tests if we do a FAR for the conditional fact.

Reply:
Or in the maybe someday case where we change desktop framework.

---
In reply to: [171398475](https://github.com/dotnet/roslyn/pull/25050#discussion_r171398475) [](ancestors = 171398475)

Reply:
FAR won't miss this. FAR is actually multi-target aware and will show separate results for every target framework in a project. 

At the same time as I said in the other file I'm a bit torn on this. For now I'm just going to use ConditionalFact everywhere. 


---
In reply to: [171422115](https://github.com/dotnet/roslyn/pull/25050#discussion_r171422115) [](ancestors = 171422115,171398637,171398475)